We deliver an older version of cryptography for Python 3.7, which seems
to be fully compatible with this version of paramiko.

Not intended for upstream.

--- paramiko-3.4.0/setup.py
+++ paramiko-3.4.0/setup.py
@@ -88,7 +88,7 @@ setup(
     python_requires=">=3.6",
     install_requires=[
         "bcrypt>=3.2",
-        "cryptography>=3.3",
+        "cryptography>=3.1",
     ],
     extras_require=extras_require,
 )
